### PR TITLE
fix: demandandcapacitynotification_swaggerui

### PR DESCRIPTION
**Pull request by eschrewe**
## Description
- added full sample request for demandandcapacitynotification request endpoint
- solves #487 

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files
- [x] If helm chart has been changed, the chart version has been bumped to either next major, minor or patch level (compared to released chart).

### CODE REVIEW

File: backend/src/main/java/org/eclipse/tractusx/puris/backend/demandandcapacitynotification/controller/DemandAndCapacityNotificationRequestApiController.java [hunk 1]
```diff
@@ -57,17 +60,19 @@ public class DemandAndCapacityNotificationRequestApiController {
     @PostMapping("request")
     public ResponseEntity<DemandAndCapacityNotificationSamm> postDemandAndCapacityNotification(
         @RequestHeader("edc-bpn") String bpnl,
-        @RequestBody String body)
+        @io.swagger.v3.oas.annotations.parameters.RequestBody(content = {@Content(examples = {
```
Comment (tom-rm-meyer-ISST) [inline backend/src/main/java/org/eclipse/tractusx/puris/backend/demandandcapacitynotification/controller/DemandAndCapacityNotificationRequestApiController.java:63]: Not sure if it was like this before but currently the swagger api always has the same example also as return type. At least for the valid response it's a null object. for the badRequest I think it's the same. Would make sense from my point to go either way:
- return the entity / body we received for each response code
- make the swagger ui respond values as currently used

Likely i would just go for the first, but feel free to go what you think would be correct (standard currently does not require this).

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
Thanks for your contribution. Issue is solved. While checking the swagger ui, I recognized one inconsistency I would be grateful if you could fix it, too :)

File: backend/src/main/java/org/eclipse/tractusx/puris/backend/demandandcapacitynotification/controller/DemandAndCapacityNotificationRequestApiController.java (hunk 1 above)
Comment (eschrewe) [reply backend/src/main/java/org/eclipse/tractusx/puris/backend/demandandcapacitynotification/controller/DemandAndCapacityNotificationRequestApiController.java:63]: It was like that before. 

The api returns no response body in any of the error cases, just the HTTP status code. And I think that's what I would show on the swagger ui. 

BTW there are several other endpoints where the situation is currently the same (like for example the production-request-api-controller and others).

Comment (tom-rm-meyer-ISST) [reply backend/src/main/java/org/eclipse/tractusx/puris/backend/demandandcapacitynotification/controller/DemandAndCapacityNotificationRequestApiController.java:63]: Fair, thanks for fixing here. Raised further issue #515 to also work on the other controllers accordingly.

**Review by tom-rm-meyer-ISST — APPROVED** [review]
Thanks a lot!

**Pull request merged**